exchanges/order/limits.go

```diff
@@ -94,26 +94,26 @@ func (e *ExecutionLimits) LoadLimits(levels []MinMaxLevel) error {
 	e.mtx.Lock()
 	defer e.mtx.Unlock()
 	if e.m == nil {
-		e.m = make(map[asset.Item]map[currency.Code]map[currency.Code]*Limits)
+		e.m = make(map[asset.Item]map[*currency.Item]map[*currency.Item]*Limits)
```

Comment:
How come you made this change?

Reply:
So the CLI could accept lowercase and uppercase currency pairs